internal/ui/promts.go

```diff
@@ -0,0 +1,57 @@
+package ui
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+type ValidateFn func(input string) error
+
+func StringPrompt(label string, validate ValidateFn) string {
+	input := ""
+	reader := bufio.NewReader(os.Stdin)
+
+	for {
+		fmt.Fprint(os.Stdout, label+": ")
+		input, _ = reader.ReadString('\n')
+		input = strings.TrimSpace(input)
+
+		err := validate(input)
+
+		if err != nil {
+			fmt.Fprintln(os.Stderr, err)
+			continue
+		}
+
+		if input != "" {
+			break
+		}
+	}
+
+	return input
+}
+
+func YesNoPrompt(label string) bool {
+	choices := "y/N"
```

Comment:
```suggestion
	choices := "y/n"
```

Reply:
`N` means it's the default behavior if left empty. Same as promptui shows it.